lib/migration.js

```diff
@@ -636,8 +636,9 @@ function mixinMigration(PostgreSQL) {
       // verify that the other model in the same DB
       if (this._models[fkEntityName]) {
         return 'CONSTRAINT ' + this.escapeName(fk.name) + ' ' +
-          'FOREIGN KEY (' + this.escapeName(fk.foreignKey) + ') ' +
-          'REFERENCES ' + this.tableEscaped(fkEntityName) + '(' + fk.entityKey + ')';
+        'FOREIGN KEY (' + this.escapeName(fk.foreignKey) + ') ' +
+        'REFERENCES ' + this.tableEscaped(fkEntityName) + '(' + fk.entityKey + ')' +
+        (fk.onDelete && fk.onDelete.toUpperCase() === 'CASCADE' ? ' ON DELETE CASCADE' : '');
```

Comment:
Have you considered adding support for other `ON DELETE` modes? Quoting from https://www.postgresql.org/docs/8.2/ddl-constraints.html:

> Restricting and cascading deletes are the two most common options. `RESTRICT` prevents deletion of a referenced row. `NO ACTION` means that if any referencing rows still exist when the constraint is checked, an error is raised; this is the default behavior if you do not specify anything. (The essential difference between these two choices is that `NO ACTION` allows the check to be deferred until later in the transaction, whereas `RESTRICT` does not.) `CASCADE` specifies that when a referenced row is deleted, row(s) referencing it should be automatically deleted as well. There are two other options: `SET NULL` and `SET DEFAULT`. These cause the referencing columns to be set to nulls or default values, respectively, when the referenced row is deleted. Note that these do not excuse you from observing any constraints. For example, if an action specifies `SET DEFAULT` but the default value would not satisfy the foreign key, the operation will fail.